docs/static/releasenotes.asciidoc

```diff
@@ -32,6 +33,79 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-9-1]]
+=== Logstash 7.9.1 Release Notes
+
+
+==== Performance improvements and notable issues fixed
+
+===== Fixed performance regression during pipeline compilation
+
+As more and more users adopt our docker images, we've been getting reports on gaps where these images don't provide all the configuration knobs the other artifacts do. This release exposes more environment variables to configure proxy support and certificate verification mode for monitoring and central management https://github.com/elastic/logstash/pull/12151[#12151], https://github.com/elastic/logstash/pull/12201[#12201] and also the configuration of the verification_mode and https://github.com/elastic/logstash/pull/12162[#12162]
+
+===== Pipeline execution fixes to flushing and shutdown
+
+Since 7.2.0, a change caused terminating pipelines to not request input plugins to cleanup before shutdown, which could cause leaks in resources that weren't freed during pipeline reloads. You can find more information on the fix here: https://github.com/elastic/logstash/pull/12195[#12195]
+
+Logstash supports ordered execution for pipelines with a single worker. A bug was found in this mode where the flushing mechanism wasn't working, preventing plugins like the aggregate filter from working correctly. This has been fixed, and you can read the details here: https://github.com/elastic/logstash/pull/12204[#12204]
+
+===== Consistent Fingerprinting
+
+Our fingerprint filter is a popular solution to perform deduplication of data in downstream systems like Elasticsearch, by computing a hash value based on data from each event. Users reported that this filter could produced different values for events containing the same data since it didn't ensure the order in which Hash Maps/Objects/Ruby Hashes processed their key/value pairs.
+This has now been fixed, and you can read more about how it was solved and all the tests we've done here: https://github.com/logstash-plugins/logstash-filter-fingerprint/pull/55[#55]
+
+===== Updated JRuby to 9.2.13.0
+
+The new JRuby release brings greater stability to its code optimizations in multithreaded workloads and a fix to exception handling in Windows environments, both issues that could affect our users. For more on the JRuby release notes, see https://github.com/jruby/jruby/releases/tag/9.2.13.0[here].
```

Comment:
```suggestion
The new JRuby release brings greater stability to its code optimizations in multithreaded workloads and a fix to exception handling in Windows environments, both issues that could affect our users. See the https://github.com/jruby/jruby/releases/tag/9.2.13.0[JRuby release notes] for more information.
```

Reply:
We try to avoid using `[here]` as link text (for improved SEO and ADA compliance).